assets/pages/regular-contributions/components/directDebitPaymentMethodDetails.jsx

```diff
@@ -0,0 +1,49 @@
+// @flow
+
+// ----- Imports ----- //
+
+import React from 'react';
+
+import PageSection from 'components/pageSection/pageSection';
+import { SvgDirectDebitSymbolAndText } from 'components/svg/svg';
+
+type PropTypes = {
+  accountHolderName: string,
+  accountNumber: string,
+  sortCodeArray: string[],
+}
+
+function mask(s: string): string {
+  return `******${s.substring(6)}`;
+}
+
+export default function DirectDebitPaymentMethodDetails(props: PropTypes) {
+  return (
+    <div className="component-direct-debit-confirmation">
+      <PageSection modifierClass="email-confirmation" heading="Your contribution">
+        <SvgDirectDebitSymbolAndText />
+        <p className="component-email-confirmation__copy">
+          Look out for an email confirming your recurring payment.
+        </p>
+        <ul className="component-direct-debit-confirmation__details">
+          <li className="component-direct-debit-confirmation__item">
+            <div className="component-direct-debit-confirmation__item-name">Payment Method:</div>
+            <div className="component-direct-debit-confirmation__item-value">Direct Debit</div>
+          </li>
```

Comment:
This structure is repeated here, it could become an aux component:

```jsx
function directDebitItem(props: {name: string, value: string}) {
  return(
    <li className="component-direct-debit-confirmation__item">
      <div className="component-direct-debit-confirmation__item-name">{props.name}</div>
      <div className="component-direct-debit-confirmation__item-value">{props.value}</div>
    </li>
  );
}
```

called four times:

```jsx
<DirectDebitItem name="Payment Method:" value="Direct Debit" />
<DirectDebitItem name="Account Name:" value={props.accountHolderName} />
<DirectDebitItem name="Account number:" value={mask(props.accountNumber)} />
<DirectDebitItem name="Sort Code:" value={{props.sortCodeArray[0]}-{props.sortCodeArray[1]}-{props.sortCodeArray[2]}} />
```